strapi/strapi
# [DEITS] import command should throw jsonl parse errors

### What does it do?

Causes the jsonl parser to report errors instead of silently ignoring them

### Why is it needed?

The file source must throw errors when trying to parse jsonl, otherwise an import may encounter an invalid line and silently continue without reporting that many entries may be missing, resulting in a "successful" import that is missing data.

### How to test it?

Create an export file that contains an unparseable .jsonl file and attempt to import it.

Prior to this PR, it would proceed silently (or in the case of schemas, give a mismatched schema validation error because data after the unparseable line would not be read)

With this PR, you should recieve an error explaining the problem and the transfer shoud halt instead of silently proceeding.
